mlflow/R/mlflow/R/install.R

```diff
@@ -20,8 +20,12 @@ mlflow_maybe_create_conda_env <- function() {
 
 #' Install MLflow
 #'
-#' Installs auxiliary dependencies of MLflow (e.g. the MLflow CLI). As a one-time setup step, you
-#' must run install_mlflow() to install these dependencies before calling other MLflow APIs.
+#' Installs auxiliary dependencies of MLflow (e.g. the MLflow CLI). As a
+#' one-time setup step, you must run install_mlflow() to install these
+#' dependencies before calling other MLflow APIs. Alternatively, you can set MLFLOW_PYTHON_BIN and
+#' MLFLOW_BIN environment variables instead. These variables allow you to use custom mlflow
```

Comment:
Since the previous statement is still the recommended way of doing things, maybe we can put this as more of a sidenote below the example. Something like, "To use a custom MLflow installation, set the MLFLOW_PYTHON_BIN and MLFLOW_BIN environment variables."

Reply:
Yeah that makes sense.